src/OpenLoco/src/World/CompanyAi.cpp

```diff
@@ -154,6 +155,35 @@ namespace OpenLoco
         2,
     };
 
+    struct ThoughtMinMaxVehicles
+    {
+        uint8_t min;
+        uint8_t max;
+    };
+    // 0x004FE784 & 0x004FE785
+    static constexpr std::array<ThoughtMinMaxVehicles, kAiThoughtTypeCount> kThoughtTypeMinMaxNumVehicles = std::to_array<ThoughtMinMaxVehicles>({
```

Comment:
Fwiw, I think you can now just do:
```suggestion
    static constexpr std::array kThoughtTypeMinMaxNumVehicles = std::to_array<ThoughtMinMaxVehicles>({
```

But I'm fine either way.

Reply:
Why not just auto?

Reply:
okay switched to auto and added a static_assert to ensure the size